helpers/github.ts

```diff
@@ -280,13 +272,13 @@ export const calculateTotalRewards = async (issues: GitHubIssue[]) => {
     }
   });
   return totalRewards;
-};
+}
 
 export async function writeTotalRewardsToGithub(totalRewards: number) {
   try {
     const owner = DEVPOOL_OWNER_NAME;
     const repo = DEVPOOL_REPO_NAME;
-    const filePath = "total_rewards.txt";
+    const filePath = "totalRewards.txt";
```

Comment:
```suggestion
    const filePath = "total-rewards.txt";
```

kebab case is like this